quick-start/src/test/java/com/marklogic/quickstart/integrationtests/IntegrationUtils.java

```diff
@@ -0,0 +1,292 @@
+package com.marklogic.quickstart.integrationtests;
+
+import static io.restassured.RestAssured.given;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+
+import org.apache.commons.lang.StringUtils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.marklogic.hub.HubTestBase;
+import com.marklogic.hub.flow.CodeFormat;
+import com.marklogic.hub.flow.DataFormat;
+import com.marklogic.hub.flow.FlowType;
+import com.marklogic.quickstart.auth.LoginInfo;
+import com.marklogic.quickstart.model.FlowModel;
+import com.marklogic.quickstart.model.JobQuery;
+
+import io.restassured.http.ContentType;
+import io.restassured.http.Cookie;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+
+public class IntegrationUtils {
+	
+	private String sessionID;
+	private String projectPath;
+	
+	// created this field to enable fetch the flowName that was created during the createFlow process
+	// and verify it.
+	private String flowName;
+	private int hmFlowNameCount = 1;
+	
+	private Cookie requestCookie;
```

Comment:
these all should be fields if this class is being used to encapsulate something.  I think though that as a utility class you probably don't want shared state and an object.  It's possible that these functions could all be static even ( not sure about that)

Reply:
I have renamed the class to E2ETestsRequestHelper.Java